backend/internal/storage/storage.go

```diff
@@ -30,6 +31,9 @@ type UserRating interface {
 	GetAllUserRatings(context.Context, uuid.UUID) ([]models.UserRating, error)
 }
 
+type Profiles interface {
+	UpdateProfilePrefences(context.Context, uuid.UUID, string, string, string, string) error
```

Comment:
This should be combined with the Profile interface, as they cover the same db table